content/en/security/notifications/rules.md

```diff
@@ -40,19 +41,20 @@ To create a notification rule, specify the conditions under which the rule shoul
 1. On the [**Notification Rules**][1] page, click **New Notification Rule**.
 1. Enter a **Name** for the notification rule.
 1. Select the source type for the notification rule:
-    - **Vulnerability**: A potential security flaw in your infrastructure.
+    - **Finding**: A potential security flaw in your infrastructure.
     - **Signal**: Suspicious activity that poses an active threat against your infrastructure.
 1. Select one or more severity levels.
 1. Specify the tags and attributes that must be present in order for the notification rule to be triggered.
+1. If you selected **Finding** in step 3, select the frequency of the notifications:
+   - **Aggregate results over**: Select this option, followed by a time frame from the list, to only get one notification for detections that occurred over that time frame.
+   - **Trigger immediately for each individual issue meeting the criteria**: Select this option to get one notification for each detection.<br />**Note**: Selecting this option can result in a large number of notifications.
 1. Click **Add Recipient**.
 1. Specify the recipients you want to notify when the notification rule is triggered. You can notify individuals, teams, create Jira issues, and more. See [Notification channels][2] for more information.
-1. To test notifications for this rule, click ** Test Notifications**. **Note: This is not available for Cloud Security Vulnerabilities.
+1. If you chose **Signal** in step 3, to test notifications for this rule, click **Test Notifications**.</br >**Note**: Testing notifications isn't available yet for finding notifications.
```

Comment:
Nit: "yet" is a temporal word. Consider removing it.

Reply:
Great catch, I'll do that - thank you!